### PR TITLE
improve doc format

### PR DESCRIPTION
**Pull request by Wesley-Du**

### CODE REVIEW

File: docs/docs/gettingstarted.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This Getting Started document provides quick reference information regarding ins
 ---
 
 
-## **1. Try Analytics Zoo**
```
Comment (hkvision) [inline docs/docs/gettingstarted.md:8]: We should keep these 1/2 unchanged.

File: docs/docs/gettingstarted.md [hunk 2]
```diff
@@ -16,71 +16,71 @@ Please also check the [examples](examples.md) for various Analytics Zoo features
 ---
 
 
-## **2. Install Analytics Zoo**
+## Install Analytics Zoo
 
 Analytics Zoo installation methods are available for Python and Scala users. 
 
-### 2.1 Python
```
Comment (hkvision) [inline docs/docs/gettingstarted.md:23]: The subsections should remove `###` and change to **2.1 Python**

File: docs/docs/gettingstarted.md (hunk 1 above)
Comment (Wesley-Du) [reply docs/docs/gettingstarted.md:8]: keep 1/2 unchanged

File: docs/docs/gettingstarted.md (hunk 2 above)
Comment (Wesley-Du) [reply docs/docs/gettingstarted.md:23]: change to 2.1 Python

File: docs/docs/gettingstarted.md (hunk 1 above)
Comment (hkvision) [reply docs/docs/gettingstarted.md:8]: ** ** is missing

File: docs/docs/gettingstarted.md (hunk 2 above)
Comment (hkvision) [reply docs/docs/gettingstarted.md:23]: missing ** ** surround 2.1 Python

File: docs/docs/gettingstarted.md (hunk 1 above)
Comment (Wesley-Du) [reply docs/docs/gettingstarted.md:8]: check again?

**Pull request merged**